server/pujan.env

```diff
@@ -0,0 +1 @@
+NAME = Pujan Soni
```

Comment:
The name of this file should be `sample.env` and there is more environment variables than just the name.